dev-utils/webdriver.js

```diff
@@ -102,6 +106,59 @@ function isLogEntryATestFailure(entry, whitelist) {
   return result
 }
 
+function getWebdriveBaseConfig(path) {
```

Comment:
nit: `getWebdriverConfig` feels better.

Reply:
I guess it supposed to be the "base config" so that it can be used as a base for the actual config. Not sure though.